services/ui-src/src/components/menus/Sidebar.tsx

```diff
@@ -1,190 +1,251 @@
 import React, { useState } from "react";
-import { Box, Button, Flex, Link, Text, FlexProps } from "@chakra-ui/react";
+import { Link as RouterLink, useLocation } from "react-router-dom";
+import { Box, Collapse, Flex, Heading, Link, Text } from "@chakra-ui/react";
+// TODO: swap out for new assets from design
 import { ArrowIcon, CheckCircleIcon } from "@cmsgov/design-system";
 
 interface LinkItemProps {
   name: string;
   path: string;
-  hasSubItems?: boolean;
-  parent?: string;
+  children?: LinkItemProps[] | null;
 }
 
 const LinkItems: LinkItemProps[] = [
-  { name: "Get Started", path: "get-started" },
   {
-    name: "A: Program Information",
-    path: "program-information",
-    hasSubItems: true,
-  },
-  {
-    name: "Point of Contact",
-    path: "point-of-contact",
-    parent: "A: Program Information",
-  },
-  {
-    name: "Reporting Period",
-    path: "reporting-period",
-    parent: "A: Program Information",
+    name: "Get Started",
+    path: "/get-started",
   },
-  { name: "Add Plans", path: "add-plans", parent: "A: Program Information" },
   {
-    name: "Add BSS Entities",
-    path: "add-bss-entities",
-    parent: "A: Program Information",
+    name: "A: Program Information",
+    path: "/program-information",
+    children: [
```

Comment:
Nested everything back up under a `children` key. You were right in thinking nesting and recursion was the way to go.